package.json

```diff
@@ -1,48 +1,54 @@
 {
-  "name": "@signalk/freeboard-sk",
-  "version": "0.0.4",
-  "description": "Openlayers chartplotter implementation for Signal K",
-  "main": "index.js",
-  "dependencies": {},
-  "browser": {
-    "bootstrap-drawer": "./node_modules/bootstrap-drawer/dist/js/drawer.js",
-    "bootstrap-slider": "./node_modules/bootstrap-slider/js/bootstrap-slider.js",
-    "bootstrap-toggle": "./node_modules/bootstrap-toggle/js/bootstrap-toggle.js",
-    "bootstrap-select": "./node_modules/bootstrap-select/js/bootstrap-select.js"
-  },
-  "devDependencies": {
-    "@signalk/client": "^0.1.4",
-    "bluebird": "^2.9.24",
-    "bootstrap": "^3.3.4",
-    "bootstrap-drawer": "^1.0.0",
-    "bootstrap-select": "^1.6.3",
-    "bootstrap-slider": "^4.8.0",
-    "bootstrap-toggle": "^2.0.0",
-    "browser": "^0.2.6",
-    "browserify": "^9.0.8",
-    "debug": "^2.1.3",
-    "font-awesome": "^4.6.3",
-    "jquery": "^2.1.3",
-    "karma": "^0.13.15",
-    "knob": "^0.3.0",
-    "lodash": "^4.17.4",
-    "mocha": "^2.3.4",
-    "nanomodal": "^5.1.0",
-    "openlayers": "^3.16.0",
-    "simplify-js": "^1.2.1",
-    "uglify-js": "^2.4.20",
-    "watchify": "^3.1.2",
-    "web-audio-daw": "^2.3.1",
-    "xml2js": "^0.4.16"
-  },
+  "name": "freeboard",
+  "version": "0.1.0",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "watchify index.js --exclude mdns --ignore bufferutil --ignore utf-8-validate --outfile bundle.js",
-    "build": "browserify index.js --exclude mdns --ignore bufferutil --ignore utf-8-validate -o bundle.js"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
   },
-  "author": "robert@42.co.nz",
-  "license": "Apache v2",
-  "keywords": [
-    "signalk-webapp"
```

Comment:
Keyword is a must here.

Reply:
Folder restructured to move angular project source to the `source` folder and the deployable files are located in the `root` .npmignore added to exclude `/source`
Application `package.json` file is now in the root folder.